ct1/vehinhtron.cpp

```diff
@@ -0,0 +1,79 @@
+#include<iostream>
+
+using namespace std;
+
+string bankinh, toadox, toadoy;
+ int x, y, r;
+
+int check(int a, int b){
+	int tmp;
+	tmp = (a-x)*(a-x)+ (b-y)*(b-y) - r*r;
+	if(tmp<=0) return 1;
+	return 0;	
+}
+void in(){
+	int i =x; int j =y;
+	for(int i=0; i<=x+r; i++){
```

Comment:
ở trên khai báo i, j mang giá trị khác, đến khi for lại gán lại về 0, vậy có ý nghĩa gì?